pkg/rtc/participant.go

```diff
@@ -50,6 +50,12 @@ type downTrackState struct {
 	downTrack   sfu.DownTrackState
 }
 
+type participantUpdateInfo struct {
+	version   uint32
+	state     livekit.ParticipantInfo_State
+	updatedAt time.Time
```

Comment:
Is it possible to use `TimedVersion` here? Would be good to standardise on that. That should contain the time information also.

Reply:
https://github.com/livekit/protocol/blob/c0705dbb696a47cf28ca946a7909fd6990decaae/utils/timed_version.go#L44

Reply:
I think it can be used here but can't find the benefits to replace the time.Time since we need to compare it with another timestamp (timestamp of last signal message received)

Reply:
You can get time using this - https://github.com/livekit/protocol/blob/c0705dbb696a47cf28ca946a7909fd6990decaae/utils/timed_version.go#L143.

Yeah, probably not strictly needed. But, trying to see if we can standardise more.

Reply:
Got it, but it need converts to compare them just for the standardize purpose, I will see if migration case need the timedversion in the cloud environment, otherwise, I prefer to keep time.Time here.